modin/core/dataframe/pandas/partitioning/partition_manager.py

```diff
@@ -1581,16 +1581,18 @@ def shuffle_partitions(
         # Convert our list of block partitions to row partitions. We need to create full-axis
         # row partitions since we need to send the whole partition to the split step as otherwise
         # we wouldn't know how to split the block partitions that don't contain the shuffling key.
-        row_partitions = [
-            partition.force_materialization().list_of_block_partitions[0]
-            for partition in cls.row_partitions(partitions)
-        ]
+        row_partitions = [partition for partition in cls.row_partitions(partitions)]
```

Comment:
```suggestion
        row_partitions = cls.row_partitions(partitions)
```
I think the list comprehension isn't really doing anything anymore, so we can probably just remove it!

Reply:
Indeed, removed it